src/test/migrations/steps/20250401111202-step-update-country-columns.ts

```diff
@@ -24,7 +24,8 @@ export default async (client: BaseProtocol) => {
                 add column last_in_editing timestamptz,
                 add column last_in_review timestamptz,
                 add column last_in_approval timestamptz,
-                add column last_in_accepted timestamptz
+                add column last_in_accepted timestamptz,
+                add column last_in_published timestamptz
                 ;`
       )
```

Comment:
I simply added the migration step to the previous migration as it was one row, but the migration keeps failing (more on DM)

Any suggestions?

It cant find the last_in_published, must be something to do with how we run the tx?

Anyway, I will proceed with other tasks tomorrow related to this and _make it work locally_ so I can get back to this whenever

-- 🌦️